internal/util/tftemplate.go

```diff
@@ -43,6 +43,11 @@ output "artifacts_bucket_name" {
 	description = "S3 bucket for artifacts like AWS Lambda code"
 	value = module.dce.artifacts_bucket_name
 }
+
+output "api_url" {
```

Comment:
I wish there was a way to render this file, without having to redeclare all the inputs and outputs....
I'll have to mull on that one...